tests/helpers/start-app.js

```diff
@@ -2,15 +2,14 @@ import Ember from 'ember';
 import Application from '../../app';
 import config from '../../config/environment';
 
-const merge = Ember.assign || Ember.merge;
-
+const { merge, run } = Ember;
```

Comment:
because the tests need to work with older ember and newer ember, the merge = assign || merge style here is preferred, and an Ignore for the suave error